sources/osgViewer/webgl-utils.js

```diff
@@ -110,7 +115,7 @@ var WebGLUtils = function () {
         /** function:(msg) */
         opt_onError ) {
         function handleCreationError( msg ) {
-            if ( msg.indexOf( 'WebGL2' ) !== -1 ) return;
+            if ( !doReportCreationError ) return;
```

Comment:
To be clear the intention is that the message is NOT displayed when trying to get a Webgl2 context.
We only display the message if we cannot get a Webgl1 context